sdk/typescript/src/builder/Commands.ts

```diff
@@ -133,42 +133,53 @@ export const Commands = {
       typeArguments?: string[];
     },
   ): MoveCallCommand {
-    return {
-      kind: 'MoveCall',
-      target: input.target,
-      arguments: input.arguments ?? [],
-      typeArguments: input.typeArguments ?? [],
-    };
+    return create(
```

Comment:
Just adding a note here, this change will validate inputs to these functions against the schema at runtime, which removes the `TODO` a little lower and also gives us higher confidence that this works correctly.